deepinv/training/trainer.py

```diff
@@ -15,6 +15,7 @@
 from deepinv.utils.plotting import prepare_images
 from torchvision.utils import save_image
 import inspect
+from deepinv.models import ArtifactRemoval, Denoiser
 
 
 @dataclass
```

Comment:
Please can you add a small note to the docstring about `model` that if you pass in a denoiser, it automatically gets converted to an `ArtifactRemoval` for training?